src/securejoin.rs

```diff
@@ -1323,12 +1333,9 @@ mod tests {
             .await
             .unwrap();
 
-        // Step 2: Bob scans QR-code, sends vg-request; blocks on ongoing process
-        let joiner = {
-            let qr = qr.clone();
-            let ctx = bob.ctx.clone();
-            async_std::task::spawn(async move { dc_join_securejoin(&ctx, &qr).await.unwrap() })
-        };
```

Comment:
i think, there was also a possible race if  `pop_sent_msg()` got executed before the first message actually was sent by `dc_join_securejoin()`. but anyway, this code part is now serialized as `dc_join_securejoin()` returns immediately also for group joins since #2508.